helm/funcx_endpoint/templates/endpoint-instance-config.yaml

```diff
@@ -20,9 +20,10 @@ data:
             HighThroughputExecutor(
                 max_workers_per_node={{ .Values.maxWorkersPerPod }},
                 address=address_by_route(),
-                strategy=KubeSimpleStrategy(max_idletime=3600),
+                strategy=KubeSimpleStrategy(max_idletime={{ .Values.maxIdleTime | default 3600 }}),
```

Comment:
I think I would rely on the default setting in `values.yaml` instead of having this default appear twice in the code

Reply:
Makes sense, I'll switch it over to the values.yaml file.